### PR TITLE
Added document for event bubbling and capturing for the `Dropdown` component.

### LINKED ISSUE #1811: Document about Tippy onClick event bubbling and capturing
https://www.loom.com/share/neetoUI-Document-about-Tippy-onClick-event-bubbling-and-capturing-13-September-2-5e18e4f967914a7d9e90147b65cc3200

@ajmaln _a

### PR DESCRIPTION
**Pull request by AkhilXavier95**
- Fixes #1811 

**Description**
- Added: Document for event bubbling and capturing for the `Dropdown` component.

**Checklist**

~- [ ] I have made corresponding changes to the documentation.~
~- [ ] I have updated the types definition of modified exports.~
~- [ ] I have verified the functionality in some of the neeto web-apps.~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
~- [ ] I have added the necessary label (patch/minor/major - If package publish
      is required).~

**Reviewers**
@ajmaln

### CODE REVIEW

**Comment by AkhilXavier95** [comment]
@ajmaln _a please review this PR.

**Review by ajmaln — APPROVED** [review]

File: stories/Components/Dropdown.stories.jsx [hunk 1]
```diff
@@ -353,6 +355,38 @@ const CustomDropdown = args => {
...
+          {listItems.map((item, idx) => (
+            <MenuItem.Button key={idx}>{item}</MenuItem.Button>
+          ))}
+          <MenuItem.Button isActive>Active</MenuItem.Button>
+          <MenuItem.Button isDisabled>Disabled</MenuItem.Button>
+          <Divider />
+          <MenuItem.Button style="danger">Delete</MenuItem.Button>
+        </Menu>
+      </Dropdown>
+    </div>
```
Comment (ajmaln) [inline stories/Components/Dropdown.stories.jsx:375]: Let's update the story with a container (like a card) that has an onClick handler, to demonstrate that the container onClick event handler is not triggered on clicking the dropdown

**Review by ajmaln — COMMENTED** [review]
@AkhilXavier95 _a Please check a minor suggestion

**Comment by AkhilXavier95** [comment]
@ajmaln _a updated the example

**Review by ajmaln — APPROVED** [review]

**Pull request merged**